tests/unit/reconcile/test_runner.py

```diff
@@ -371,4 +371,78 @@ def test_run_with_failed_execution():
     recon_runner = ReconcileRunner(ws, installation, install_state, prompts)
     with pytest.raises(SystemExit):
         recon_runner.run()
-    ws.jobs.run_now.assert_called_once_with(1234)
+    ws.jobs.run_now.assert_called_once_with(1234, job_parameters={'operation_name': 'reconcile'})
+
+
+def test_aggregates_reconcile_run_with_job_id_in_state(monkeypatch):
+    monkeypatch.setattr("webbrowser.open", lambda url: None)
+    ws = create_autospec(WorkspaceClient)
+    prompts = MockPrompts(
+        {
+            r"Would you like to open the job run URL .*": "yes",
+        }
+    )
+    installation = MockInstallation(
+        {
```

Comment:
can you move `state` and `reconcile.yml` as variables

Reply:
Done